src/components/questions/IntegerQuestion.vue

```diff
@@ -167,6 +167,9 @@ export default {
       }
     },
     question() {
+      if (this.selected === "") {
+        this.$refs.integerInput.value = "";
```

Comment:
I now understand why this is necessary. Please add another null check for integerInput ($refs is never null) and add a comment explaining why this is needed.